app/Http/Controllers/HomeController.php

```diff
@@ -145,12 +145,13 @@ public function search()
         ]);
 
         $search = new Search($params);
+        $isSearchPage = true;
```

Comment:
seems like search should be its own section at this point ┐(°～° )┌

Reply:
There's a whole new search page design coming in, so  ┐(°～° )┌